src/content/operations/deployment/subctl/_index.en.md

```diff
@@ -328,6 +328,108 @@ It is recommended to use this when reporting any issue.
 
 `subctl gather --module broker,service-discovery --type resources`
 
+### `cloud`
+
+#### `cloud prepare`
+
+`subctl cloud prepare [flags]`
+
+This command prepares the cloud for Submariner installation.
+
+#### `prepare aws` flags
+
+This command prepares an AWS-based cloud for Submariner installation.
```

Comment:
Is it really an "AWS cloud" or OpenShift/IPI on AWS?

Reply:
Thats a good point. We have to update the description in the subctl operator too as this text is copied from the help section.